### ISSUE TITLE
RDF::Literals shouldn't cast to strings by default.

### ISSUE THREAD

**Issue by tpendragon** [ActiveTriples/ActiveTriples#244]
It seems friendly, but all it does is lose data.

**Comment by no-reply** [comment]
:+1:

Currently:

```ruby
require 'rdf/vocab'

class Book
  include ActiveTriples::RDFSource  
  property :title, predicate: RDF::Vocab::DC.title  
end  

book = Book.new

book.title << RDF::Literal("Moomin", language: :en)
book.title << RDF::Literal("Mummi",  language: :fi)

book.title = book.title
book.dump :ttl
# => "\n [ <http://purl.org/dc/terms/title> \"Mummi\",\n       \"Moomin\"] .\n"
```

**Comment by tpendragon** [comment]
I think casting from literals with language tags to a string was a bad idea. Functionality should be something like this:

```ruby
require 'rdf/vocab'
class Book
  include ActiveTriples::RDFSource
  property :title, predicate: RDF::Vocab::DC.title
end

book = Book.new
book.title << "Moomin"
book.title << RDF::Literal("Moomin")
book.title << RDF::Literal("Moomin", language: :en)
book.title << RDF::Literal("Mummi", language: :fi)

book.title # => ["Moomin", RDF::Literal("Moomin", language: :en), RDF::Literal("Mummi", language: :fi)]
```

**Comment by no-reply** [comment]
Closed by #246.
